infra/github-runner/prod/container_app_environment.tf

```diff
@@ -0,0 +1,27 @@
+resource "azurerm_container_app_environment" "github_runner" {
+  name                = "${local.project}-runner-cae-01"
+  location            = local.location
+  resource_group_name = azurerm_resource_group.github_runner.name
+
+  log_analytics_workspace_id = data.azurerm_log_analytics_workspace.log.id
+
+  infrastructure_subnet_id       = azurerm_subnet.github_runner.id
+  zone_redundancy_enabled        = false
+  internal_load_balancer_enabled = true
+
+  workload_profile {
+    name                  = "Consumption"
+    workload_profile_type = "Consumption"
+    minimum_count         = 0
+    maximum_count         = 0
```

Comment:
why 0?

Reply:
Because their values are not used by the consumption profile, and the ARM silently put them to 0 underneath. However, they're mandatory values in TF so they must be explicit